index.md

```diff
@@ -52,3 +192,31 @@ which most in the community are members of.
 If you have technical questions around DNS flag day you can join
 [the DNS-operations mailing list](https://lists.dns-oarc.net/mailman/listinfo/dns-operations)
 and ask them there.
+
+Get in touch
+============
+
+For press & media inquiries please use media (at) dns-oarc.net and please put
+“DNS Flag Day" in the email subject line.
+
+- Web https://dnsflagday.net/
+- Twitter https://twitter.com/dnsflagday
+- Announcements: https://lists.dns-oarc.net/mailman/listinfo/dns-announce
+- Discussion: https://lists.dns-oarc.net/mailman/listinfo/dns-operations
+
+FAQ
+===
+
+- Q: Is DNS over UDP dead?
+
+  A: No, UDP will still be the main means of transportation.
+
+- Q: TL;DR [RFC 7766](https://tools.ietf.org/html/rfc7766)
+
+  A: DNS **MUST** work over TCP!
+
+- Q: Will everything break on date-to-be-decided 2020?
+
+  A: **No!** The date that will be announced is just a marker in time from
+     which the default behavior _**in new software releases**_ will be
+     changed.
```

Comment:
- Q: Why is TCP support so important?

  A: TCP support should be implemented from the beginning of DNS, although some may have taken the early specification to mean that the transport protocol is optional. Hence ten years ago RFC 5966 made TCP support mandatory. Blocking TCP, or failure to support TCP, may result in resolution failure and application-level timeouts. 
  
  In addition, TCP can be operated in a mode that does not require IP fragmentation, and makes it harder to spoof DNS responses.

- Q: Why not just switch to TCP only?

  A: DNS over UDP is fine for small packets that do not require IP fragmentation. It can still be used for that class of DNS messages, which is the larger part of the Internet traffic. Switching everything over on TCP may cause stress on DNS services. While in principle DNS over TCP only should be feasable, it is slower than DNS over UDP, in the best case by a factor of 4, and it may limit the number of connections a DNS server can accept simultaneously.

- Q: Will this Flag Day require a software update, or is it only a configuration change?

  A: All open source DNS servers already support TCP and respect the configured EDNS buffer sizes. At most, DNS operators will have to change their configuration as described in "How to test?"